client/src/components/Applications/AppForm/AppFormSchema.js

```diff
@@ -2,131 +2,111 @@ import * as Yup from 'yup';
 
 const FormSchema = (app) => {
   const appFields = {
-    parameters: {},
-    inputs: {},
-    defaults: { inputs: {}, parameters: {} },
-    schema: { inputs: {}, parameters: {} },
+    appArgs: {},
+    envVariables: {},
+    fileInputs: {},
+    defaults: { fileInputs: {}, appArgs: {} },
+    schema: { fileInputs: {}, appArgs: {} },
   };
-
-  (app.definition.parameters || []).forEach((parameter) => {
-    const param = parameter;
-    if (!param.value.visible || param.id.startsWith('_')) {
+  /* TODOv3  envVariables */
+  (app.definition.jobAttributes.parameterSet.appArgs || []).forEach((p) => {
+    const param = p;
+    // TODOv3 should we rename 'visible' to 'hidden' so that we default to showing argument
```

Comment:
we decided to move from `visible` to `isHIdden`

https://docs.google.com/spreadsheets/d/1asrCkdcB1FR_LI-cZj6vxBc4qwpvccDHZUgijz41llE/edit#gid=0